### PR TITLE
Add the ability to specify adhoc oc commands

### PR DESCRIPTION
**Pull request by firemanxbr**
* looking to support the CVP project in "Post Deployment Tasks"
* fixing the issue #33

### CODE REVIEW

**Comment by scoheb** [comment]
What you have submitted are not adhoc commands but rather specific commands that are for our team only. This was not the nature of the issue.

**Comment by firemanxbr** [comment]
@scoheb I created a test import_playbook from parameter but are failing the module "import_playbook" [1] 

[1] - https://gist.githubusercontent.com/firemanxbr/457954be5ccc34a696363d34a3a6e83c/raw/578230d59050796b6e518dbbe4c77c00b49a93c6/playbooks_hooks_error

**Comment by firemanxbr** [comment]
Right now we have the resource to add other playbooks to run in the last playbook that will be run on contra-env-setup. 
@scoheb 
@arilivigni

File: README.md [hunk 1]
```diff
@@ -18,12 +18,14 @@
   - [Project repo options that has s2i templates and Jenkins Pipelines](#project-repo-options-that-has-s2i-templates-and-jenkins-pipelines)
     - [OpenShift s2i template setup options](#openshift-s2i-template-setup-options)
     - [Jenkins 2.0 pipeline setup options](#jenkins-20-pipeline-setup-options)
+  - [Templates](#templates)
   - [Usage examples](#usage-examples)
-    - [Example 1: Setup on a local server :: Setup Minishift + OS templates](#example-1-setup-on-a-local-server--setup-minishift--os-templates)
-    - [Example 2: Setup on a local server :: Setup Minishift + OS templates + Jenkins 2.0 pipelines](#example-2-setup-on-a-local-server--setup-minishift--os-templates--jenkins-20-pipelines)
-    - [Example 3: Setup on a local server :: Setup Minishift + OS templates + Jenkins 2.0 pipelines](#example-3-setup-on-a-local-server--setup-minishift--os-templates--jenkins-20-pipelines)
+    - [Example 1: Setup on a local machine :: Setup Minishift + OS templates](#example-1--setup-on-a-local-machine----setup-minishift-os-templates)
```
Comment (arilivigni) [inline README.md]: Did you use doctoc to generate a new TOC?

File: playbooks/group_vars/all/global.yml [hunk 2]
```diff
@@ -6,6 +6,7 @@ setup_minishift: true
 setup_containers: true
 setup_pipelines: false
 setup_sample_project: false
+setup_playbook_hooks: true
```
Comment (arilivigni) [inline playbooks/group_vars/all/global.yml]: Shouldn't this be false by default?

File: README.md (hunk 1 above)
Comment (firemanxbr) [reply README.md]: @arilivigni no, I edit using vim.

Comment (arilivigni) [reply README.md]: please use markdown-toclify to generate the TOC.

**Comment by arilivigni** [comment]
@scoheb can you review please

**Comment by scoheb** [comment]
testing now...

**Comment by arilivigni** [comment]
@scoheb cut a v1.1.0 release?

**Pull request merged**